content/300-accelerate/350-static-ip.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: "Static IP"
+metaTitle: "Enable Static IP for Prisma Accelerate"
+metaDescription: "Learn enabling Static IP for Prisma Accelerate."
+---
+
+You can enable static IP for Accelerate when your security setup requires IP allowlisting or if you're implementing firewalls that only permit access from trusted IPs, ensuring controlled and secure database connections.
+
+![](./images/result-of-adding-static-ip-to-accelerate.png)
```

Comment:
Make sure to move images into our `static/img` directory. You can see an example of how images are used here: https://github.com/prisma/docs/blob/26e4ac09ca997da279bec50ad98b75e71889cc6d/content/200-orm/200-prisma-client/100-queries/055-pagination.mdx?plain=1#L108